kubernetes/deployment-staging-azure-canary.tmpl

```diff
@@ -43,6 +43,10 @@ spec:
             value: '65000'
           - name: STORAGE_ADAPTER
             value: azure
+          - name:  STORAGE_URL
+            value: 'https://panoptes.blob.core.windows.net/staging'
```

Comment:
```suggestion
            value: 'https://panoptes.blob.core.windows.net'
```

Reply:
the actual storage account container URL in the azure portal is `https://panoptes.blob.core.windows.net/public` so it's the `storage-account.domain/container-name-path`

Not sure where I got staging from tbh. I think it needs to change to `https://panoptes.blob.core.windows.net/public` instead. What do you think?